tests/index.rs

```diff
@@ -35,3 +35,32 @@ fn re_opening_database_does_not_trigger_schema_check() {
     .rpc_server(&rpc_server)
     .run();
 }
+
+#[test]
+fn index_runs_with_rpc_user_and_pass_as_env_vars() {
```

Comment:
Let's modify this test to make sure we're using the correct values. Let's record the user and pass which are passed each time a client connects, and then do something like `assert_eq!(rpc_server.client_credentials, &[("foo", "bar")])`.

Reply:
The way the our mock bitcoin core instance is written I do not have access to the Basic Authentication http header. So I can't know what credentials are passed in.